tests/OAuth2ClientHandler.Tests/AuthorizerTests.cs

```diff
@@ -144,6 +162,26 @@ public void GetToken_ClientCredentialsWithScope_ShouldRequestScope()
         }
 
         [Test]
+        public async Task GetToken_ValidPasswordCredentials_FormCredentials_ReturnsValidAccessToken()
+        {
+            var options = new AuthorizerOptions
+            {
+                AuthorizeEndpointUrl = new Uri("http://localhost/authorize"),
+                TokenEndpointUrl = new Uri("http://localhost/token"),
+                ClientId = "MyId",
+                ClientSecret = "MySecret",
+                Username = "MyUsername",
+                Password = "MyPassword",
+                GrantType = GrantType.ResourceOwnerPasswordCredentials,
+                CredentialsType = CredentialsType.Form
+            };
+
+            var authorizer = new Authorizer.Authorizer(options, () => server.HttpClient);
+            var result = await authorizer.GetToken();
+            Assert.NotNull(result.AccessToken);
+        }
+
```

Comment:
Possible to add an other test case with an invalid secret while using the forms credential type?